ansible/playbooks/sap-hana-cluster.yaml

```diff
@@ -40,6 +43,11 @@
     - name: Load SAP HANA variables
       ansible.builtin.include_vars: ./vars/hana_vars.yaml
```

Comment:
This file has `ansible.builtin.include_vars: ./vars/hana_vars.yaml` :green_circle: